Catappult/appcoins-sdk-ios
# Feature/app 2820 query user currency

**What does this PR do?**

   This PR adds a new method called getUserCurrency to CurrencyUseCases and CurrencyRepository. With this getUserCurrency we'll call convertCurrency service on AppCoinBillingService to obtain the user currency. The received data on CurrencyRepository will be stored in memory.

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] CurrencyUseCases.swift
- [ ] CurrencyRepository.swift
- [ ] AppCoinBillingService.swift

**How should this be manually tested?**

  Call CurrencyUseCases.shared.getUserCurrency on AppcSDKInternal and verify if we are receiving the required data.

**What are the relevant tickets?**

  Tickets related to this pull-request: APP-2820

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new keys, etc.) 

**Code Review Checklist**
